website/pages/_document.js

```diff
@@ -9,8 +9,25 @@ class MyDocument extends Document {
   render() {
     return (
       <Html lang="en">
-        <Head />
+        <Head>
+          <script
```

Comment:
When we'd like to add GTM/ GA we must also add a way for the user to consent the data collection (via a cookie banner e.g.). Without it would be a violation of the GDPR.